.sync.yml

```diff
@@ -42,12 +42,38 @@
     services: docker
     sudo: required
     stage: acceptance
+  - bundler_args: 
+    dist: trusty
+    env: PLATFORMS=ub_puppet5
+    rvm: 2.5.3
+    before_script:
+    - bundle exec rake 'litmus:provision_list[waffle_ub]'
```

Comment:
Hey @david22swan ,

This should be updated to travis_ub.

Cheers!

Reply:
fixed